src/mono/mono/mini/mini.c

```diff
@@ -4345,6 +4345,11 @@ mini_get_cpu_features (MonoCompile* cfg)
 	}
 #endif
 
+#if defined(TARGET_ARM64)
+	// All Arm64 devices have this set
+	features |= MONO_CPU_ARM64_BASE; 
+#endif
```

Comment:
Why is this needed if its always set ?

Reply:
At least for CoreCLR, we support disabling this via `COMPlus_EnableHWIntrinsic=0` so that you can easily test software fallbacks, etc.

Reply:
Yes, it makes it possible to disable ArmBase via `mattr=-base` if needed